oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -153,13 +153,14 @@ public class VersionGarbageCollector {
     VersionGarbageCollector(DocumentNodeStore nodeStore,
                             VersionGCSupport gcSupport,
                             final boolean detailedGCEnabled,
-                            final boolean isDetailedGCDryRun) {
+                            final boolean isDetailedGCDryRun,
+                            final boolean embeddedVerification) {
         this.nodeStore = nodeStore;
         this.versionStore = gcSupport;
         this.ds = gcSupport.getDocumentStore();
         this.detailedGCEnabled = detailedGCEnabled;
         this.isDetailedGCDryRun = isDetailedGCDryRun;
-        this.embeddedVerification = true; //TODO: make this "configurable"
+        this.embeddedVerification = embeddedVerification; //TODO: make this "configurable" for actual detailedGC
```

Comment:
detail: this TODO could now also move to DocumentNodeStore (assuming it will have to be adjusted)

Reply:
The only reason, I didn't move this to `DocumentNodeStore` is because of the https://issues.apache.org/jira/browse/OAK-10633 ticket. This ticket would eventually remove this TODO once finished.